contracts/MicroLotto.sol

```diff
@@ -98,8 +98,11 @@ contract MicroLotto {
     function ownerCollect() public {
```

Comment:
`accumulatedValue` is resetted in `draw()` and blocks the funds forever. Split `accumulatedValue` to separate fields `prize` and `fees`.